ail_vru_uc1_scenario/docker-compose.yml

```diff
@@ -61,8 +60,7 @@ services:
       - ROS_IP=172.3.0.3
       - ROS_MASTER_URI=http://172.3.0.2:11311/
     command: >
-      bash -c 'export RMW_IMPLEMENTATION=rmw_fastrtps_cpp
-      && wait-for-it.sh 172.3.0.2:11311 -- roslaunch /opt/carma/vehicle/config/carma_docker.launch'
+      bash -c 'export RMW_IMPLEMENTATION=rmw_fastrtps_cpp && wait-for-it.sh 172.3.0.2:11311 -- roslaunch /opt/carma/vehicle/config/carma_docker.launch'
```

Comment:
If you are using some sort of code format and it is reformatting all these file by default, I think it would be a good idea to try and include in our documented development process. Otherwise you will constantly be making some random formatting changes that others with different format settings or none will be constantly undoing. It makes git blame impossible to read. For the future can we not include these formatting changes until we have a formatting guideline.

Reply:
yeah this was a mistake and oversight. Not sure how it ended up there. I started from clean and now includes only the required changes.